### ISSUE TITLE
flatpak unable to install

### ISSUE THREAD

**Issue by pyamsoft** [flathub/com.jetbrains.WebStorm#88]
Hello,

I am unable to install/update to the latest version (from Flathub) because of the following error message upon install:

```bash
Error: While trying to apply extra data: apply_extra script failed, exit status 256
```

As a result, the user is unable to install or update to the latest commit

Flatpak 1.15.6

**Comment by x80486** [comment]
I think this is an issue on your end. Check your DNS and connectivity in general. Moreover, verify you can download this: `https://download.jetbrains.com/webstorm/WebStorm-2023.2.5.tar.gz`.

I just installed the latest version (several times, for that matter) and it went fine every single time — these are the logs from the first time.

```shell
x80486@ubuntu-vm:~$ flatpak install flathub com.jetbrains.WebStorm 
Looking for matches\u2026
Remote \u2018flathub\u2019 found in multiple installations:

   1) system
   2) user

Which do you want to use (0 to abort)? [0-2]: 
Which do you want to use (0 to abort)? [0-2]: 1
Required runtime for com.jetbrains.WebStorm/x86_64/stable (runtime/org.freedesktop.Sdk/x86_64/23.08) found in remote flathub
Do you want to install it? [Y/n]: 

com.jetbrains.WebStorm permissions:
    ipc                  network       wayland      x11      dri      file access [1]
    dbus access [2]      tags [3]

    [1] host, xdg-run/keyring
    [2] org.freedesktop.Flatpak, org.freedesktop.Notifications, org.freedesktop.secrets
    [3] proprietary


        ID                                             Branch                 Op            Remote             Download
 1. [\u2713] org.freedesktop.Platform.GL.default            23.08                  i             flathub            163.4\u00a0MB / 163.0\u00a0MB
 2. [\u2713] org.freedesktop.Platform.GL.default            23.08-extra            i             flathub             17.8\u00a0MB / 163.0\u00a0MB
 3. [\u2713] org.freedesktop.Sdk.Locale                     23.08                  i             flathub             18.2\u00a0kB / 361.7\u00a0MB
 4. [\u2713] org.freedesktop.Sdk                            23.08                  i             flathub            523.8\u00a0MB / 555.1\u00a0MB
 5. [\u2713] com.jetbrains.WebStorm                         stable                 i             flathub            606.5\u00a0MB / 606.2\u00a0MB

Installation complete.
```

**Comment by pyamsoft** [comment]
Hello

Yes I believe this is a "my system" configuration specific issue.

For record, I was able to perform the download just fine - the failure was a file system permission issue on the unpacking step when running the `apply_extra` script which expected to untar an archive.

I was installing the flatpak into the priv `system` repo, but running it as an unpriv normal `user`

I solved the issue by installing the flatpak into the `user` repo and running as the same user.
